src/category_theory/abelian/exact.lean

```diff
@@ -330,9 +331,13 @@ namespace functor
 open limits abelian
 
 variables {A : Type u₁} {B : Type u₂} [category.{v₁} A] [category.{v₂} B]
-variables [has_zero_object A] [has_zero_morphisms A] [has_images A] [has_equalizers A]
+variables [abelian A]
```

Comment:
I preferred the previous typeclass requirements, since it becomes apparent that the result follows with very few limits and without assuming a preadditive structure. It's fine to allow `abelian A` for the newly added results.